bootstrap/README.md

```diff
@@ -77,3 +84,59 @@ cargo run --release --example compute_hashes -- --file ./canister_state.bin > ch
 ```
 
 The hash of each chunk is saved in `chunk_hashes.txt`.
+
+## 7. Hardcode the caches into the uploader canister
+
+```
+cp bootstrap/chunk_hashes.txt bootstrap/uploader/src/chunk_hashes.txt
+```
+
+Update the `state_size` in `bootstrap/uploader/src/main.rs` to be equal to the state size in 64k blocks, i.e. `echo $(( $(stat -c %s bootstrap/canister_state.bin) / 65536 ))`
+
+## 8. Deploy the uploader canister
+
+`dfx` version `0.12.1` is known to work
+
+Add desired network to the `dfx.json` and set `NETWORK` to match it.
```

Comment:
The dfx.json is already at version 0.13.1. Why go back to 0.12.1?

Reply:
0.13 doesn't create canisters on testnets